### PR TITLE
Space cat breathes space

### LINKED ISSUE #20152: The space cat suffocates in space
## Description
**Reproduction**
- Spawn a space cat, in space
- The space cat dies in space
- The space cat died in space

image
space cat gasping in space

### PR DESCRIPTION
**Pull request by TexCruize**
## About the PR

Made it so that the Space Cat no longer suffocates in space.
Fixes #20152

## Why / Balance

To make the Space Cat not cry as its space suit actually works (kinda)

## Media

image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

None
**Changelog**

:cl: TomCruize
- fix: Made Space Cats lungs ROBUST (They now don't suffocate in space)

### CODE REVIEW

**Review by LankLTE — APPROVED** [review]
based

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 1]
```diff
@@ -2074,19 +2074,48 @@
...
+  parent: SimpleSpaceMobBase
   description: Feline pet, prepared for the worst.
   components:
   - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Pets/cat.rsi
     layers:
     - map: ["enum.DamageStateVisualLayers.Base"]
       state: spacecat
+  - type: Physics
```
Comment (Doru991) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:2086]: Turning it into a space mob makes the prototype a bit bloated, in my opinion simply 'disabling' the respirator component by stopping its saturation level from going down would result in less unnecessary copy-pasting.
```suggestion
  - type: Respirator
    minSaturation: 5.0
```

**Review by Doru991 — CHANGES_REQUESTED** [review]
Turning it into a space mob makes the prototype a bit bloated, in my opinion simply 'disabling' the respirator component by stopping its saturation level from going down would result in less unnecessary copy-pasting.

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml (hunk 1 above)
Comment (TexCruize) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: I have made your changes and tested them, thank you for your help. o7

**Review by Doru991 — APPROVED** [review]
While I still think they should have to use internals so it's more realistic, I don't think there's a way to automatically turn on a mob's internals yet so this is good enough

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml (hunk 1 above)
Comment (Doru991) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: You're welcome, this was the first commit I felt competent enough to launch a review on o7

**Pull request merged**